qdb/utils.py

```diff
@@ -186,7 +198,11 @@ def __call__(self, seconds, exception=None, green=False):
 
     ast.Global,
     ast.Pass,
-}
+
+    # Python 2 only
+    ast.Print if PY2 else None,
```

Comment:
It seemed like a good way to make a frozenset while filtering them out, do you think this would cause any strange behaviour later?